README.adoc

```diff
@@ -97,6 +110,9 @@ public class BindingInitializer {
         .setUsername("myuser") // <2>
         .setPassword("mypassword"); // <3>
         .setSecure(true) // <4>
+        .setConnectTimeout(10000) // <5>
+        .setConnectRetryWaitTime(10000) // <6>
+        .setRequestedConnectionHeartbeatTimeout(3) // <7>
```

Comment:
Problem here if you try to set 500 milliseconds what to do? Library only accepts seconds

Reply:
given you had `unit`as an TimeUnit and `value' you can do the following internally:

`seconds = unit.toSeconds(value)`

Reply:
Sure but if you set timeunit to milliseconds and say 50 milliseconds 
Like:
setRequestedConnectionHeartbeatTimeout(TimeUnit.MILLISECONDS, 50)
Which would result in 0.05 seconds represented as integer seconds would be zero. so instead of waiting 50 ms it would wait 0 s. Thats all i mean. So i think calling this method with anything below 1 sec should throw an exception unless you call it explicitly with zero which is on some timouts indefinitly i think on setTimout it is described in rabbitmq javadoc that zero has this effect there. I dont know if it is valid for heartbeat i dont think so

Reply:
We will need to check the resulting time boundaries in any case here. Even when we use integer values one could set a negative value here... I suggest to throw an IllegalArgumentException in those cases...

Reply:
Yeah thats true, ok i will add this change